content/200-concepts/050-overview/100-what-is-prisma/100-data-modeling.mdx

```diff
@@ -154,12 +154,12 @@ The resulting `user` object is an instance of Sequelize's `Model` class (because
 
 ## Data modeling with Prisma
 
-Depending on which parts of Prisma you want to use in your application, the data modeling flow looks slightly different. The following two sections explain the workflows for using [**only Prisma Client**](#using-only-prisma-client) and using [**Prisma Client and Prisma Migrate (Experimental)**](#using-prisma-client-and-prisma-migrate-experimental).
+Depending on which parts of Prisma you want to use in your application, the data modeling flow looks slightly different. The following two sections explain the workflows for using [**only Prisma Client**](#using-only-prisma-client) and using [**Prisma Client and Prisma Migrate (Preview)**](#using-prisma-client-and-prisma-migrate-experimental).
```

Comment:
Might it be that the anchor link would go from `#using-prisma-client-and-prisma-migrate-experimental` to `#using-prisma-client-and-prisma-migrate-preview` ?